src/python/pants/base/fingerprint_strategy.py

```diff
@@ -78,6 +79,9 @@ class TaskIdentityFingerprintStrategy(FingerprintStrategy):
   :API: public
   """
 
+  @deprecated('1.5.0.dev0',
+              'The information that was previously included in the fingerprint by '
+              'TaskIdentityFingerprintStrategy is now included by default in all Tasks.')
```

Comment:
How about suggesting the replacement since presumably some of the importing tasks would presumably be using using TaskIdentity as a super class (mine do).

I guess the new default is just the DefaultFingerprintStrategy now, although I notice you didn't port forward the TaskIdentityFingerprintStrategy consumers in this commit.